source/unified-test-format/unified-test-format.rst

```diff
@@ -0,0 +1,2546 @@
+record whether or not each issue is addressed by this spec.
+
+The following tickets are addressed by the test format:
+
+* `SPEC-1158 <https://jira.mongodb.org/browse/SPEC-1158>`__: Spec tests should use a consistent format for CRUD options
+
+  ``options`` structs are now flattened into `operation.arguments`_. See:
+  `Entity Test Operations`_.
+
+* `SPEC-1215 <https://jira.mongodb.org/browse/SPEC-1215>`__: Introduce spec test syntax for meta assertions (e.g. any value, not exists)
+
+  See: `Special Operators for Matching Assertions`_
+
+* `SPEC-1216 <https://jira.mongodb.org/browse/SPEC-1216>`__: Update GridFS YAML tests to use newer format
+
+  See: `stream <entity_stream_>`_ entity and `bucket`_ operations
+
+* `SPEC-1229 <https://jira.mongodb.org/browse/SPEC-1229>`__: Standardize spec-test syntax for topology assertions
+
+  See `runOnRequirement`_, which is used by both `runOnRequirements`_ and
+  `test.runOnRequirements`_.
+
+* `SPEC-1254 <https://jira.mongodb.org/browse/SPEC-1254>`__: Rename topology field in spec tests to topologies
+
+  See `runOnRequirement`_.
+
+* `SPEC-1439 <https://jira.mongodb.org/browse/SPEC-1439>`__: Inconsistent error checking in spec tests
+
+  This may still require test format syntax to assert that an error occurred
+  without caring about any assertions on the error itself.
+
+* `SPEC-1713 <https://jira.mongodb.org/browse/SPEC-1713>`__: Allow runOn to be defined per-test in addition to per-file
+
+  See `runOnRequirements`_ and `test.runOnRequirements`_.
+
+* `SPEC-1723 <https://jira.mongodb.org/browse/SPEC-1723>`__: Introduce test file syntax to disable dropping of collection under test
+
+  Only collections in `initialData`_ are dropped, so this can be achieved by
+  omitting the collection from `initialData`_. Additionally, the format supports
+  creating an empty collection without inserting any documents (needed by
+  transaction tests).
+
+The following tickets can be resolved after the unified test format is approved
+and/or other specs begin porting their tests to the format:
+
+* `SPEC-1102 <https://jira.mongodb.org/browse/SPEC-1102>`__: Add "object: collection" to command monitoring tests
+* `SPEC-1133 <https://jira.mongodb.org/browse/SPEC-1133>`__: Use APM to assert outgoing commands in CRUD spec tests
+* `SPEC-1144 <https://jira.mongodb.org/browse/SPEC-1144>`__: CRUD tests improvements
+* `SPEC-1193 <https://jira.mongodb.org/browse/SPEC-1193>`__: Convert change stream spec tests to runOn format
+* `SPEC-1230 <https://jira.mongodb.org/browse/SPEC-1230>`__: Rewrite APM spec tests to use common test format
+* `SPEC-1238 <https://jira.mongodb.org/browse/SPEC-1238>`__: Convert retryable write spec tests to multi-operation format
+* `SPEC-1261 <https://jira.mongodb.org/browse/SPEC-1261>`__: Use runOn syntax to specify APM test requirements
+* `SPEC-1375 <https://jira.mongodb.org/browse/SPEC-1375>`__: changeStream spec tests should be run on sharded clusters
+
+
+Future Work
```

Comment:
is it worth adding mention of adding support for logging expectations in the future here?

Reply:
Added in most recent commit. Please resolve when acknowledged.

Reply:
LGTM, thanks.